### DISCUSSION TITLE
On The Way

### DISCUSSION THREAD

**Discussion by akgembi12** [sonsuzus/BilsemPython#72] [Genel]
"On The Way" takımına girmek isteyenler buraya adını, gününü yazabilir

Ben: Ali Ekrem KAYLI-ÖYG2
salı günü son 2 ders

**Comment by TahaGuzell** [comment]
Merhaba Taha Güzel oyg2 Perşembe 3. ve 4. dersler

**Reply by akgembi12** [reply]
takıma alındın : )
numaranı söylersen talegram gurubuna ekleyeceğim

**Comment by MamiA11** [comment]
Muhammed Ali Nas
ÖYG-2
Salı 5-6. Dersler

**Reply by akgembi12** [reply]
takıma alındın : )
numaranı söylersen talegram gurubuna ekleyeceğim

**Comment by Musabkaracinar** [comment]
Musab Karaçınar
ÖYG-2
Salı 5-6. Dersler

**Reply by akgembi12** [reply]
takıma alındın : )
numaranı söylersen talegram gurubuna ekleyeceğim

**Comment by Ameth28** [comment]
Ahmet Emre Dağ
ÖYG - 2
Salı 5 - 6. Dersler

**Reply by akgembi12** [reply]
takıma alındın : )
numaranı söylersen talegram gurubuna ekleyeceğim

**Comment by Atakanoyg1** [comment]
merhaba katılmak istiyorum adım Atakan ŞAHİN ÖYG 1 Çarşamba 3. ve 4. dersler

**Comment by Ardemin** [comment]
Merhaba ben Arın Demir İnan "On The Way" takımına katılmak istiyorum,
Öyg-2
salı günü 3 ve 4. derslere giriyorum.

**Comment by Muhammedipek** [comment]
Ben Muhammed İpek Öyg 2 öğrencisiyim perşembe son iki ders grubundayım takıma yararlı olacağımı düşünüyorum

**Comment by akgembi12** [comment]
Takımda olanlar bu linkten gruba katılsınlar

https://t.me/+E_2hgjM57-dmMWNk

**Comment by akgembi12** [comment]
Takımda olanlar bu linkten gruba katılsınlar

https://t.me/+E_2hgjM57-dmMWNk
